src/js/outcome.js

```diff
@@ -39,16 +39,26 @@ class Position {
   }
   trial(display_element, trial) {
     const show_circle = () => {
+      // check if make prediction (determines what feedback receive)
+      var predicExist = this.jsPsych.data.get().select('prediction').values.slice(-1)[0];
```

Comment:
Is the goal to prevent the shield and picker from from rendering at a null position?

Reply:
yes so if there's no response/null, there'll be no shield/picker